README.md

```diff
@@ -1,10 +1,12 @@
 # Terminus Composer Plugin
 
 [![CircleCI](https://circleci.com/gh/pantheon-systems/terminus-composer-plugin.svg?style=shield)](https://circleci.com/gh/pantheon-systems/terminus-composer-plugin)
+[![Terminus v2.x Compatible](https://img.shields.io/badge/terminus-v2.x-green.svg)](https://github.com/pantheon-systems/terminus-composer-plugin/tree/1.x)
+[![Terminus v3.x Compatible](https://img.shields.io/badge/terminus-v3.x-green.svg)](https://github.com/pantheon-systems/terminus-composer-plugin/tree/1.x)
```

Comment:
Let's follow the pattern in https://github.com/pantheon-systems/terminus-plugin-example

Reply:
Done.